src/pages/Login/Login.page.jsx

```diff
@@ -1,39 +1,40 @@
 import React from 'react';
 import { useHistory } from 'react-router';
-
 import { useAuth } from '../../providers/Auth';
 import './Login.styles.css';
 
-function LoginPage() {
+export default function LoginPage() {
   const { login } = useAuth();
   const history = useHistory();
 
-  function authenticate(event) {
-    event.preventDefault();
-    login();
-    history.push('/secret');
-  }
+  const authenticate = (e) => {
+    e.preventDefault();
+
+    const name = document.getElementById('username').value;
+    const password = document.getElementById('password').value;
```

Comment:
You are obtaining the values using vanillajs, I recommend to use controlled inputs.
For example:
```js
const [name, setName] = useState('').
....


<input required type="text" value={name} onChange={(event) => setName(event.target.value)}/>
```